templates/layout.html

```diff
@@ -40,8 +40,7 @@ <h1>
         <a href={{ url_for('logout') }}>Logout</a>
       </nav>
     {% else %}
-      <a href={{ url_for('login') }} class=login>Log in</a>
-      <a href={{ url_for('signup') }} class=login>Sign up</a>
```

Comment:
when an user is not logged he could be want to sign up or log in? WDYT?

Reply:
I put it back